compiler_gym/third_party/LexedIR/decoy_main.cpp

```diff
@@ -0,0 +1,35 @@
+#include "HughsLexer.h"
```

Comment:
Is this file needed?

Reply:
Nope, it's a test main that I never removed. Deleting...